.gitlab/jobs/tioga.yml

```diff
@@ -43,16 +43,27 @@ setup:
 
 # Build and test jobs, simply provide a spec
 
-#rocm_5_7_1:
-#  extends: .job_on_tioga
-#  stage: jobs-stage-2
-#  variables:
-#    SPEC: "+rocm amdgpu_target=gfx90a %rocmcc@=5.7.1 ^hip@5.7.1"
-#    THREADS: 12
-
-cce_16_0_1:
+.tioga_job_command:
+  script:
+    - echo PROXY="${PROXY}"
+    - echo TIOGA_JOB_ALLOC="${TIOGA_JOB_ALLOC}"
+    - "printf '#!/bin/bash\n%s\n' \"${JOB_CMD}\" > flux_script.sh"
+    - cat flux_script.sh
+    - ${PROXY} flux watch $( ${PROXY} flux batch -o output.stdout.type=kvs ${TIOGA_JOB_ALLOC} flux_script.sh )
+    - rm -f flux_script.sh
```

Comment:
Without these changes, `SPEC` strings with spaces did not work on Tioga. We may want to move this to the `radiuss/radiuss-shared-ci` repo.

Reply:
Whitespaces _and quotes_ are perfectly supported in my other projects. I am looking for the root cause of this.

Reply:
Note that this workaround was needed just on Tioga, the other machines work fine (they have different job commands).

Reply:
`SPEC` is passed as en environment variable in my other projects, which is more robust to "nesting". I’d like to find a solution that does not involve on-the-fly script generation.

Reply:
More accurately: I would like to avoid overriding `.tioga_job_command`